public/app/features/browse-dashboards/RecentlyDeletedPage.tsx

```diff
@@ -33,33 +35,50 @@ const RecentlyDeletedPage = memo(() => {
     );
   }, [dispatch, stateManager]);
 
+  if (searchState.loading === false && searchState.result && searchState.result.totalRows === 0) {
```

Comment:
```suggestion
  if (searchState.loading === false && searchState.result?.totalRows === 0) {
```
I think you can simplify this a bit.